src/views/Rankings.vue

```diff
@@ -121,6 +121,87 @@
         </v-card>
       </v-menu>
       <v-card-text>
+        <div class="custom-table-wrapper elevation-1">
```

Comment:
This is already extracted in the rankings-grid component. Guess you had merge conflicts. The whole table needs to be removed.